WultraMobileTokenSDK/Operations/QR/WMTQROperation.swift

```diff
@@ -52,7 +55,10 @@ public struct WMTQROperation {
     }
     
     internal var dataForOfflineSigning: Data {
-        return "\(operationId)&\(operationData.sourceString)".data(using: .utf8)!
+        guard let totp = totp else {
```

Comment:
This is a bad use of guard, `if let` is more suitable. Guard should be used when the rest of the function does not make sense. This is just 2 different paths.